dashboard/src/components/Ideas.vue

```diff
@@ -0,0 +1,45 @@
+<!--
```

Comment:
I feel like the Ideas page should be a redirect instead of an iFrame

Reply:
![image](https://user-images.githubusercontent.com/6048348/86511402-1381c700-bdf9-11ea-85fa-1ae0ed98ae8f.png)